src/main/java/me/contaria/seedqueue/customization/Layout.java

```diff
@@ -71,8 +73,8 @@ private static Layout fromJson(JsonObject jsonObject) throws JsonParseException
                 Group.fromJson(jsonObject.getAsJsonObject("main"), SeedQueue.config.rows, SeedQueue.config.columns),
                 jsonObject.has("locked") ? Group.fromJson(jsonObject.getAsJsonObject("locked")) : null,
                 jsonObject.has("preparing") ? Group.fromJson(jsonObject.getAsJsonArray("preparing")) : new Group[0],
-                jsonObject.has("replaceLockedInstances") && jsonObject.get("replaceLockedInstances").getAsBoolean()
-        );
+                jsonObject.has("replaceLockedInstances") && jsonObject.get("replaceLockedInstances").getAsBoolean(),
+                jsonObject.has("mainFillOrder") ? MainFillOrder.valueOf(jsonObject.get("mainFillOrder").getAsString().toUpperCase()) : MainFillOrder.FORWARD);
```

Comment:
`String#toUpperCase` is sensitive to localization, please use `toUpperCase(Locale.ROOT)` instead